tests/functional/workloads/cnv/test_cnv_node_replacement.py

```diff
@@ -0,0 +1,140 @@
+import logging
+import pytest
+import random
+
+from ocs_ci.framework import config
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.framework.pytest_customization.marks import workloads, magenta_squad
+from ocs_ci.helpers.cnv_helpers import cal_md5sum_vm, run_dd_io
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
+from tests.functional.z_cluster.nodes.test_node_replacement_proactive import (
+    delete_and_create_osd_node,
+    select_osd_node_name,
+)
+from ocs_ci.helpers.helpers import (
+    verify_storagecluster_nodetopology,
+)
+from ocs_ci.helpers.sanity_helpers import Sanity
+
+logger = logging.getLogger(__name__)
+
+
+@magenta_squad
+@workloads
+class TestCnvNodeReplace(E2ETest):
+    """
+    Node replacement proactive
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def setup(self, project_factory, multi_cnv_workload):
+        """
+        Setting up VMs for tests
+
+        """
+
+        # Create a project
+        proj_obj = project_factory()
+        (
+            self.vm_objs_def,
+            self.vm_objs_aggr,
+            self.sc_obj_def_compr,
+            self.sc_obj_aggressive,
+        ) = multi_cnv_workload(namespace=proj_obj.namespace)
+
+        logger.info("All vms created successfully")
+
+    def test_vms_with_node_replacement(
+        self,
+        setup_cnv,
+        setup,
+    ):
+        """
+        Node Replacement proactive
+        """
+        all_vms = self.vm_objs_def + self.vm_objs_aggr
+
+        file_paths = ["/source_file.txt", "/new_file.txt"]
```

Comment:
make sure that you run some continuous IO from the VMs throughout the node replacement procedure

Reply:
PR 11624 will take care of this.